public/app/services/user-service.ts

```diff
@@ -0,0 +1,25 @@
+import {Injectable}     from 'angular2/core';
+import {Observable}     from 'rxjs/Observable';
+import {Http, Response} from 'angular2/http';
+import {User}           from './user';
+
+
```

Comment:
More boilerplate angular2 tutorial code. See the HttpClient section of the dev guide.